include/swift/AST/DiagnosticsSIL.def

```diff
@@ -342,6 +342,21 @@ WARNING(warning_int_to_fp_inexact, none,
   "'%1' is not exactly representable as %0; it becomes '%2'",
   (Type, StringRef, StringRef))
 
+
+// Yield usage errors
+ERROR(return_before_yield, none, "accessor doesn't 'yield' before returning",())
+
+ERROR(multiple_yields, none, "encountered multiple 'yield's along "
+  "a single path in yield_once accessor", ())
+
+NOTE(previous_yield, none, "previous 'yield' was here", ())
+
+ERROR(possible_return_before_yield, none, "cannot guarantee that the accessor "
+  "'yield's a value before returning", ())
```

Comment:
This probably reads better if you just don't quote `yield`.  It's a verb, like `return`, and doesn't need to be treated as super-technical jargon.